### DISCUSSION TITLE
Actions not running for PRs made by another workflow

### DISCUSSION THREAD

**Discussion by vadi2** [community/community#26800] [Actions]
<p>I have a scheduled workflow that creates PRs as needed - however, none of the checks are running on it. Why? Because the checks are required, these PRs are effectively un-mergeable.</p>
<p><a href="https://github.com/Mudlet/Mudlet/pull/5447" rel="noopener nofollow ugc">Example PR</a>.</p>

**Comment by vadi2** [comment]
<p>This is adding a bit of unnecessary maintainer load because to merge these PRs one has to disable the branch protection checks every time. Is this something that can be fixed?</p>

**Comment by airtower-luna** [comment]
<p>How are those PRs created? If the <code>GITHUB_TOKEN</code> is used for authentication <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow">it won’t trigger workflows by design</a>. If you use a PAT workflows should be running just like when you’re doing it manually.</p>

**Comment by vadi2** [comment]
<p>I see I can also give a special <a href="https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#permissions" rel="noopener nofollow ugc">permissions</a> here. Which one would be the correct one?</p>
<p>It’s difficult to test since if I run the workflow manually, the permissions are right.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>None of those. If you want to trigger new workflows, you have to use a PAT for authentication, not <code>GITHUB_TOKEN</code>.</p>

**Comment by vadi2** [comment]
<p>Thanks, that was it.</p>
